server/auth.go

```diff
@@ -523,6 +527,14 @@ func processUserPermissionsTemplate(lim jwt.UserPermissionLimits, ujwt *jwt.User
 	} else if lim.Permissions.Pub.Deny, err = applyTemplate(lim.Permissions.Pub.Deny, true); err != nil {
 		return jwt.UserPermissionLimits{}, err
 	}
+
+	// if pub/sub allow where not empty, but are empty post template processing, add in a deny to compensate
```

Comment:
`were` instead of `where`. But more importantly, looks like the code is opposite of the comment. That is, you wrote if they were not empty, but are empty after processing add deny. Then should it not be `if !subAllowWasEmpty && ..` (notice the "!"). Same for the pub.

Reply:
wrong variable name. fixed

Reply:
thanks for pointing that out